test/mongoose_graphql_SUITE.erl

```diff
@@ -870,73 +881,165 @@ use_dir_module_and_service_not_loaded(Config) ->
 
-use_dir_auth_admin_all_modules_and_services_loaded(Config) ->
+use_dir_auth_admin_all_modules_services_and_dbs_loaded(Config) ->
     Doc = <<"{ catC { command2 } }">>,
     Ctx = #{user => jid:make_bare(<<"admin">>, <<"localhost">>)},
     {Ast, Ctx2} = check_directives(Config, Ctx, Doc),
     Res = execute_ast(Config, Ctx2, Ast),
     ?assertEqual(#{data => #{<<"catC">> => #{<<"command2">> => <<"command2">>}}}, Res).
 
-use_dir_auth_user_module_and_service_not_loaded(Config) ->
+use_dir_auth_user_module_service_and_db_not_loaded(Config) ->
     Doc = <<"{ catB { command2 } }">>,
     Ctx = #{user => jid:make_bare(<<"user">>, <<"localhost">>)},
     {Ast, Ctx2} = check_directives(Config, Ctx, Doc),
     #{errors := [Error]} = execute_ast(Config, Ctx2, Ast),
     #{extensions :=
         #{code := deps_not_loaded,
-          not_loaded_modules := [<<"mod_x">>],
-          not_loaded_services := [<<"service_x">>]
+        not_loaded_modules := [<<"mod_x">>],
+        not_loaded_services := [<<"service_x">>],
+        not_loaded_internal_databases := [<<"db_x">>]
          },
-      message := <<"Some of required modules or services are not loaded">>,
+    message :=
+        <<"Some of the required modules and services and internal databases are not loaded">>,
       path := [<<"catB">>, <<"command2">>]
      } = Error.
 
-use_dir_auth_admin_module_and_service_not_loaded(Config) ->
+use_dir_auth_admin_module_service_and_db_not_loaded(Config) ->
     Doc = <<"{ catB { command2 } }">>,
     Ctx = #{user => jid:make_bare(<<"admin">>, <<"localhost">>)},
     {Ast, Ctx2} = check_directives(Config, Ctx, Doc),
     #{errors := [Error]} = execute_ast(Config, Ctx2, Ast),
     #{extensions :=
         #{code := deps_not_loaded,
           not_loaded_modules := [<<"mod_x">>],
-          not_loaded_services := [<<"service_x">>]
+          not_loaded_services := [<<"service_x">>],
+          not_loaded_internal_databases := [<<"db_x">>]
          },
-      message := <<"Some of required modules or services are not loaded">>,
+      message :=
+        <<"Some of the required modules and services and internal databases are not loaded">>,
       path := [<<"catB">>, <<"command2">>]
      } = Error.
 
+use_dir_auth_user_db_not_loaded(Config) ->
+    Doc = <<"{ catD { command2 } }">>,
+    Ctx = #{user => jid:make_bare(<<"user">>, <<"localhost">>)},
+    {Ast, Ctx2} = check_directives(Config, Ctx, Doc),
+    #{errors := [Error]} = execute_ast(Config, Ctx2, Ast),
+    #{extensions :=
+        #{code := deps_not_loaded,
+          not_loaded_internal_databases := [<<"db_x">>]},
+          message := <<"Some of the required internal databases are not loaded">>,
+      path := [<<"catD">>, <<"command2">>]
+     } = Error.
+
+use_dir_auth_admin_db_not_loaded(Config) ->
+    Doc = <<"{ catD { command2 } }">>,
+    Ctx = #{user => jid:make_bare(<<"admin">>, <<"localhost">>)},
+    {Ast, Ctx2} = check_directives(Config, Ctx, Doc),
```

Comment:
A bunch of lines repeated from the previous test case, maybe use a helper function?